docs/deployment.md

```diff
@@ -85,44 +84,65 @@ Port  | Protocol | Function
+---- | -------  | ---
+5775 | UDP      | accept zipkin.thrift over compact thrift protocol
+6831 | UDP      | accept jaeger.thrift over compact thrift protocol
+6832 | UDP      | accept jaeger.thrift over binary thrift protocol
+5778 | HTTP     | serve configs, sampling strategies
 
-A script is provided to initialize Cassandra keyspace and schema
-using Cassandra's interactive shell [`cqlsh`][cqlsh]:
+It can be executed directly on the host or via Docker, as follows:
 
-```sh
-MODE=test sh ./plugin/storage/cassandra/schema/create.sh | cqlsh
+```bash
+## make sure to expose only the ports you use in your deployment scenario!
+docker run \
+  --rm \
+  -p5775:5775/udp \
+  -p6831:6831/udp \
+  -p6832:6832/udp \
+  -p5778:5778/tcp \
+  jaegertracing/jaeger-agent
+  /go/bin/agent-linux --collector.host-port=jaeger-collector.jaeger-infra.svc:14267
 ```
 
-For production deployment, pass `MODE=prod DATACENTER={datacenter}` arguments to the script,
-where `{datacenter}` is the name used in the Cassandra configuration / network topology.
-
-The script also allows overriding TTL, keyspace name, replication factor, etc.
-Run the script without arguments to see the full list of recognized parameters.
+### Discovery System Integration
 
-### ElasticSearch
+The agents can connect point to point to a single collector address, which could be
+load balanced by another infrastructure component (e.g. DNS) across multiple collectors.
+The agent can also be configured with a static list of collector addresses.
 
-ElasticSearch does not require initialization other than
-[installing and running ElasticSearch](https://www.elastic.co/downloads/elasticsearch).
-Once it is running, pass the correct configuration values to the Jaeger collector and query service.
+On Docker, a command like the following can be used:
 
-#### Shards and Replicas for ElasticSearch indices
+```bash
+docker run \
+  --rm \
+  -p5775:5775/udp \
+  -p6831:6831/udp \
+  -p6832:6832/udp \
+  -p5778:5778/tcp \
+  jaegertracing/jaeger-agent \
+  /go/bin/agent-linux --collector.host-port=jaeger-collector.jaeger-infra.svc:14267
+```
 
-Shards and replicas are some configuration values to take special attention to, because this is decided upon
-index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
-more information about choosing how many shards should be chosen for optimization.
+In the future we will support different service discovery systems to dynamically load balance
+across several collectors ([issue 213](https://github.com/uber/jaeger/issues/213)).
 
 ## Query Service & UI
 
 **jaeger-query** serves the API endpoints and a React/Javascript UI.
 The service is stateless and is typically run behind a load balancer, e.g. nginx.
 
+An example to test Query Service:
+
+```
+docker run -it -p16686:16686 jaegertracing/jaeger-query:latest /go/bin/query-linux --cassandra.keyspace=jaeger_v1_test --cassandra.servers=192.168.0.183
```

Comment:
Break this one too, like my previous comment.

Reply:
Sure :+1: on it :)